modules/schema_validation.lua

```diff
@@ -4,6 +4,8 @@ local hmi_types = api.init("data/HMI_API.xml", true)
 local mob_types = api.init("data/MOBILE_API.xml")
 if (not hmi_api) then hmi_api = xml.open("data/HMI_API.xml") end
 if (not mobile_api) then mobile_api = xml.open("data/MOBILE_API.xml") end
+local wrong_function_name = "WrongFunctionName"
+local genric_response = "GenericResponse"
```

Comment:
@VVeremjova fixed in https://github.com/AMikhniev/sdl_atf/commit/78c1c75063563f13dcaa3956e9dc1709d4e3f282